docs: add code of conduct #7798 (#7)
linkorb/.github: CODE_OF_CONDUCT.md

## TL;DR

> Be nice. Provide and accept constructive feedback. Avoid spamming, abusive, trolling, and otherwise unacceptable behaviour. Repeat violations may result in a permanent ban.

Reviewer: ```suggestion
Be nice. Provide and accept constructive feedback. Avoid spamming, abusive, trolling, and otherwise unacceptable behavior. Repeat violations may result in a permanent ban.
```

## Scope

> This Code of Conduct applies to all persons, including contributors, maintainers, end users, sponsors, etc., who interract with this project. It applies within all community spaces, and also applies when

Reviewer: ```suggestion
This Code of Conduct applies to all persons, including contributors, maintainers, end users, sponsors, etc., who interact with this project. It applies within all community spaces and also applies when
```

an individual is officially representing the community in public spaces.

> Examples of representing our community include using an official e-mail address,

Reviewer: ```suggestion
Examples of representing our community include using an official email address,
```
